src/components/RegionVaccinationBlock/StateVaccinationBlock.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { Fragment } from 'react';
+import { Region } from 'common/regions';
+import Grid from '@material-ui/core/Grid';
+import { Heading2, Paragraph } from './RegionVaccinationBlock.style';
+import FeedbackBox from './FeedbackBox';
+import { StateLinkButton } from './StateVaccionationBlock.style';
+import { trackVaccinationLink } from './utils';
+
+const StateVaccinationBlock: React.FC<{ region: Region }> = ({ region }) => {
+  return (
+    <Fragment>
+      <Heading2>How to get vaccinated</Heading2>
+      <Paragraph>
+        Depending on your location, you may have to schedule an appointment or
+        get on a waitlist.
+      </Paragraph>
+      <Grid container spacing={1}>
+        <Grid item xs={12} md={6} key="eligibility">
+          <StateLinkButton
+            href=""
+            target="_blank"
+            rel="noopener noreferrer"
+            onClick={() =>
+              trackVaccinationLink(`Eligibiligy: ${region.fullName}`)
```

Comment:
typo - also is there a way to include both values? I guess we probably don't have that level of specificity, but would be nice if we were able to have the hierarchy of `[vaccination, click_link, eligibility, region_name]`.  Just curious how we're planning on aggregating this event

Reply:
No, GA only has category, action and label